tests/sles4sap/hana.pm

```diff
@@ -0,0 +1,139 @@
+# SUSE's SLES4SAP openQA tests
+#
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Checks HANA Wizard installation on Gnome
+# Requires: ENV variable MEDIA pointing to installation media, QEMURAM=32768
+# Maintainer: Ricardo Branco <rbranco@suse.de>
+
+use base 'sles4sap';
+use strict;
+use testapi;
+use utils 'turn_off_gnome_screensaver';
+use utils 'type_string_slow';
+
+sub run {
+    my ($self) = @_;
+    my ($proto, $path) = split m|://|, get_required_var('MEDIA');
+
+    die "Currently supported protocols are nfs and smb"
+      unless ($proto eq 'nfs' or $proto eq 'smb');
```

Comment:
hint: You could use `$proto =~ /nfs|smb/;` but that might be personal style preferences as well